meta-xt-prod-cockpit-rcar-driver-domain/recipes-core/images/core-image-minimal.bbappend

```diff
@@ -0,0 +1,17 @@
+
+IMAGE_INSTALL_append = " \
+    dual-render-3d-part \
+    xen-tools \
+    kernel-modules \
+    util-linux \
+"
+
+do_install[depends] += "arm-trusted-firmware:do_deploy"
+do_install[depends] += "optee-os:do_deploy"
+do_install[depends] += "u-boot:do_deploy"
```

Comment:
arm-trusted-firmware, optee-os and u-boot are added to EXTRA_IMAGEDEPENDS inside meta-renesas/.../rcar_common.inc

So, no need to make these dependencies. Binaries will be added to the image.